### DISCUSSION TITLE
关于Dns劫持污染的担忧向作者大大个各位大佬求教

### DISCUSSION THREAD

**Discussion by yjyyyyx** [kkkgo/PaoPaoDNS#240] [问答和帮助]
昨日在TG群里看到个求助，
部署在国外的中文图床网站
66img.cc域名在江苏电信宽带
以doh方式向223.5.5.5查询，
返回的却是221.228.32.13 240e:5a:2003:3131::41，
其他地区的解析确正常。
        我自己理解是：
在解析结果的返回过程中被劫持？了，
但这是以doh查询的，这是什么dns攻击技术？
paopao能多大程度抵御哪些dns攻击？
如果这种歪分邪气吹到了我这里，能、该如何防御？

**Comment by hxw600326** [comment]
遇到了同样的情况，错误解析的地址都一样221.228.32.13 240e:5a:2003:3131::41。不过我是江苏移动宽带。用阿里的dns解析，阿里自带的工具检查，发现只有江苏移动的dns解析地址全部错误，其它省的无论移动还是电信都正确的，用阿里的云主机nslookup地址也是对的。221.228.32.13 这个地址是江苏电信的，点击打开转到公安的反诈页面。怀疑是不是被公安误杀了。
